test/Rules/MessageMustBeStaticRuleTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SfpTest\PHPStan\Psr\Log\Rules;
+
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+use Sfp\PHPStan\Psr\Log\Rules\MessageMustBeStaticRule;
+use Sfp\PHPStan\Psr\Log\Rules\PlaceHolderInMessageRule;
+
+/**
+ * @implements RuleTestCase<PlaceHolderInMessageRule>
```

Comment:
```suggestion
 * @implements RuleTestCase<MessageMustBeStaticRule>
```

Please also update below `@covers`, too.